objcgen/objcgenerator-subscripts.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+using IKVM.Reflection;
+using Type = IKVM.Reflection.Type;
+
+using Embeddinator;
+
+namespace ObjC
+{
```

Comment:
Minor: If we are following [Mono Coding Guidelines](http://www.mono-project.com/community/contributing/coding-guidelines/#where-to-put-braces) then the brace is wrong